### ISSUE TITLE
ec2: unsupported parameter for module: source_dest_check

### ISSUE THREAD

**Issue by evanccnyc** [ansible/ansible#6780]
##### Issue Type:

Bug Report
##### Ansible Version:

1.5.3
##### Environment:

N/A. 
##### Summary:

No matter what I do with the "source_dest_check" parameter in ec2, it fails outputting:

> failed: [127.0.0.1] => {"failed": true, "item": ""}
> msg: unsupported parameter for module: source_dest_check
##### Steps To Reproduce:

Use code attached below, run it and get results

```
- name: Create the NAT Instance
  ec2:
    region: "{{ aws_region}}"

    aws_access_key: "{{ aws_access_key }}"
    aws_secret_key: "{{ aws_secret_key }}"
    keypair: "{{ aws_keypair }}"
    instance_type: "{{ vpc_nat_instancetype }}"
    image: "{{ vpc_nat_instance_id }}"
    vpc_subnet_id: '{{vpc["subnets"][0]["id"]}}'
    assign_public_ip: true
    group: ansibleDeployedDefault
    source_dest_check: false
    wait: true
    instance_tags: 
      Name: "NAT Instance"
    exact_count: 1
  register: ec2
```
##### Expected Results:

source_dest_check is turned off on the ec2 instance
##### Actual Results:

Error message with the following output:

```
failed: [127.0.0.1] => {"failed": true, "item": ""}
msg: unsupported parameter for module: source_dest_check
```

**Comment by mpdehaan** [comment]
This was added in 1.5, any chance you are pointed at an older module library directory?

**Comment by evanccnyc** [comment]
I have been upgrading via pip. How would I check that?

**Comment by evanccnyc** [comment]
I reinstalled ansible from scratch, still seeing the same thing.

**Comment by jctanner** [comment]
@evanccopengeo please download this gist file, execute it and paste the output into a comment:

https://gist.github.com/jctanner/10022471

**Comment by evanccnyc** [comment]
Here is the output from the command:

```
# INSTALLED LIB PATH
/Library/Python/2.7/site-packages/ansible
# OTHER INSTALLED LIB PATHS
/Library/Python/2.7/site-packages/ansible
# MODULES
/usr/share/ansible/cloud/cloudformation     3276ef237fa5e8a8bedc3908cd9c79b3e1d5a809adb9cd7658f32fca06694658
/usr/share/ansible/cloud/digital_ocean  205ec69f16a16a22d03fb1fc3921628f1c7a41060a04d830c100cc08b84c6e74
/usr/share/ansible/cloud/docker     bef90111af0f7a384f1705d03ed6b496b083b4b1eb40adbb172c0cefcca5b9f6
/usr/share/ansible/cloud/docker_image   5f024364b49d0bfe134bcb28faea7e91344afc6fb91e5f673b0afad4f6a7424a
/usr/share/ansible/cloud/ec2    dfa3eb83398d6bced17f770924fa20b08c590765cb9f73d569aedef3532fe6f0
/usr/share/ansible/cloud/ec2_ami    a07a4035f82b0e11afbf2d2e60746ac4c21358c2f0f0611dbab952ad8da0c262
/usr/share/ansible/cloud/ec2_eip    679ba3d9c44ad493a891296f614ecd1e34756b738546899a13cd07292dd50252
/usr/share/ansible/cloud/ec2_elb    99ab5a87b48b235dfc5a78f61ad7bacf34fa09e05089db4f79fcd1221bfe3928
/usr/share/ansible/cloud/ec2_elb_lb     3912cf1d93652d35954ca3516de15ca3d07b47fbe2afbfb3ac3daab9bd3c3319
/usr/share/ansible/cloud/ec2_facts  e36b32fcc5c1dc31de58b917f3a94682571203f52affffb83345b737e7d43eb0
/usr/share/ansible/cloud/ec2_group  b4ae2dae9ccd90cfa31e7f93b2aa56cd491b9d9226a9b185a5aa3b80085e9e0c
/usr/share/ansible/cloud/ec2_key    f63483b8d51d7cef47a67d0735d9423208c1b70c91465df1610faef8ca992348
/usr/share/ansible/cloud/ec2_snapshot   679d6fdb5f745068597f7ef118c93be2565bbbff1557f6b54ddaca55f7713b8a
/usr/share/ansible/cloud/ec2_tag    320af701a1cd30cc67eaefcc1da990b524461514d251b4f950befce49909f80f
/usr/share/ansible/cloud/ec2_vol    98e4c4a5e96449110d8164f244debb1b474446decb2eda6a3fcc2ebbcc9cf2a8
/usr/share/ansible/cloud/ec2_vpc    b36c64327c968f18fb0db5d7cf803ed5532b9721addfd822dd956d21bd0f4e1c
/usr/share/ansible/cloud/elasticache    32a8aac7bebbf085235217f5d79302357e5a5fb760033e8f559f93f92c55470b
/usr/share/ansible/cloud/gc_storage     779e69230ce96988970e3b73cb39b98917ce3566ee35f3a2849f40d696177f7a
/usr/share/ansible/cloud/gce    5b2aa7a5228d8e7fe8de2035ec464770a3f6c75a1f55c83fc470fbc344aae58c
/usr/share/ansible/cloud/gce_lb     c9fc4f01a6683c5678205413fa363c2b721210697563c339bacede66d759a985
/usr/share/ansible/cloud/gce_net    75099df3150c7b1fa68cdeeff2e21c5adba19310380c47a54ac0c4ef6cf03393
/usr/share/ansible/cloud/gce_pd     5b01d334f786ce0a0662d82b55a5bfe9d38e8d0be2b950876310680c09e3e9b4
/usr/share/ansible/cloud/glance_image   a8facb0c1520391bf04791d3747a8884e416a51b4e0f509fee10ae3576aa849c
/usr/share/ansible/cloud/keystone_user  e81549eb7184472713dda225fd25ab3581cb13ce6f353dc18b9085983075978e
/usr/share/ansible/cloud/linode     0de4d737c6c880c56ecc749047834a825187542d61e92f8947717c2eb99e6f56
/usr/share/ansible/cloud/nova_compute   0b1e23e5641cf27b766038ad4ff7e33eeebb0cb5ca714b1e427a3a5df708ba2b
/usr/share/ansible/cloud/nova_keypair   86591c9baea4806275db329e0912cf4135567d80c3ab5b42b8dd66a55ba23901
/usr/share/ansible/cloud/ovirt  24795bf7450e4f4ad6b91db7a1233bae2723fe0ee447bd03d9ce1ca635003c27
/usr/share/ansible/cloud/quantum_floating_ip    9c2b6116f5c4b2073eec4abe987166ff5bc19077295c50e3e5202213f52c0cf4
/usr/share/ansible/cloud/quantum_floating_ip_associate  673711a2f1c5400733b3e3e3d0abbda32bcc36d80f2fdeeaf5729b88d904285c
/usr/share/ansible/cloud/quantum_network    b409e9a6a289d731d2fac1e74f6ecd740e0af4855dadf54cc71ac33e531aa544
/usr/share/ansible/cloud/quantum_router     0b811b4ceeeb0d777b49573bd30b70af5c1264662a11bab9a4955c26d8de170b
/usr/share/ansible/cloud/quantum_router_gateway     3b4a95d359246a1536d6f1cb99a05df7c038f5721658d613d702b49066c1d305
/usr/share/ansible/cloud/quantum_router_interface   f7bec5eafa24c47016177aefe4aa238bde58881f2eb5578bc24639dfc4830867
/usr/share/ansible/cloud/quantum_subnet     01c3df0a8e55fb106e652f70cfd47001aed9d3c327f6bb6bc7e8ed72b2f354e5
/usr/share/ansible/cloud/rax    0be151fda96c051bca27eb40aa805a8f7f2e15133890f2ba5cd616219b7da5f8
/usr/share/ansible/cloud/rax_clb    7575f4c36d9fb2b57cd8eaac222f5841ed7749c3ea3ee8181a9ab9ff44e2f67d
/usr/share/ansible/cloud/rax_clb_nodes  1bbc043942dbc0f4f44590bceba686c002b4a72009dd49df9308aeab00966d6e
/usr/share/ansible/cloud/rax_dns    13f32ba7f4658714fdef4d5462444d8f0c7970f2984cd25c30f954c1be89305d
/usr/share/ansible/cloud/rax_dns_record     5885341b5f5a0d86b46f4b4322791bb5e5c3ec68a1203cb70c3e5366617d85c9
/usr/share/ansible/cloud/rax_facts  1b710eabbb1a2143fffffd56a60fad7145deb5b1341095c0fd41da35667b378d
/usr/share/ansible/cloud/rax_files  b94f7dc4c8f8e5a17ee7592e2cc333040a77be563d9df1e44f2f978e2f88428d
/usr/share/ansible/cloud/rax_files_objects  d0cc8493ca7d8dc17de107e39ca102605dac2b6c13cd7233669f1feddd45932f
/usr/share/ansible/cloud/rax_keypair    0e57b3df27021543d98322eab4b69ddcf9cc3912bde052b99bc752a20629329a
/usr/share/ansible/cloud/rax_network    256e9f981f040af3a2701eb74f2b071b37d50581193b5b59a7156323a93974f2
/usr/share/ansible/cloud/rax_queue  b86e79311083ae4b53daf64e04d7d06c3ca83b59fdf43c1a52fcf80a411b0263
/usr/share/ansible/cloud/rds    5bc0be8d86fe97351a04a54a8c7ca07f1414e48732c988323023c8e8c1ca605a
/usr/share/ansible/cloud/route53    aa7aeefd6f1be8cdca2cee7783b4ddea4e14ce8f7f50e8a887eb5f37a46c169a
/usr/share/ansible/cloud/s3     6af4c65a6819b57d6f338398ad45b41053329de5382ec3347fce25ee024c9acd
/usr/share/ansible/cloud/virt   dfa1c1eebc91f1173a2c129b2fd4d8858a6a72bdc7eda67be8aaf31b30e5827b
/usr/share/ansible/commands/command     feddcf0dafe410facbc4a2b3a6d5e29f2c67e690dced14f3115d32b70925bd8e
/usr/share/ansible/commands/raw     c1643e679d82db6dd555cfbe55370a37f919afdc99980cf090011f8f0b39c286
/usr/share/ansible/commands/script  f76304ace7fb233e32d3bb1b0ce9e5bea0cb553a88151b3a073f795c7aa25ad0
/usr/share/ansible/commands/shell   628c391ae93363ca7761b40e7a5b75909d3f36742c5239870aebd7319832c154
/usr/share/ansible/database/mongodb_user    09cfa2a226af46a4c6222de9d6bff11e81c672ee7b98c5ba909749eb2d07edfc
/usr/share/ansible/database/mysql_db    a7b888fc493b4dd38de92f4a1e2be4600f579ca798b144b6f136c47fff9e6eb9
/usr/share/ansible/database/mysql_replication   5ce84b30c0f5cda47b8830442384f593da3f3fe000db80e4b41946510b0a6de0
/usr/share/ansible/database/mysql_user  dae151b0a9e7546c6c8320b9047531d4b72dbeace8788d6321e010ab1e71b4a7
/usr/share/ansible/database/mysql_variables     ee59f238cd01700a139a84a3c9507c2b8dd1aa8bd9737faaa5da63d2861910e5
/usr/share/ansible/database/postgresql_db   c32f564da8ffa7d681e9fead5f3177d97bd66478f09073040af25df668250fda
/usr/share/ansible/database/postgresql_privs    1468521fab70e061c10ed8a5ebed170727f72f01f90711076179d481be9fd7be
/usr/share/ansible/database/postgresql_user     904beff7f2460a715e7b00e5d2abc0cf842d0c4582e170a9ef9558d9accaf133
/usr/share/ansible/database/redis   9c08968675d5b65bc91337059448786ef05ccae47bfba56808b55d63e2130485
/usr/share/ansible/database/riak    733a0d673fef08046eb7664eb89022b50aebff7c5981fc807caafc3a741ddece
/usr/share/ansible/files/acl    beb3b81e35e69ad27179b5a63c2352bc2e4dacf8ae2e839849d8a75d61210baa
/usr/share/ansible/files/assemble   4755d3b3232ee8e882e311fe7abe287afdf936fc09d9f06d7ef619d4bfa06131
/usr/share/ansible/files/copy   58180965a8dcfa9606d442be28e26262ca98ed7cdc126a4a4b19a9f10f870b6d
/usr/share/ansible/files/fetch  ea03940951ec07fc9ac3df9751ec2b95b2b63e6c1c2075c68702df176d57f0ed
/usr/share/ansible/files/file   da92cf1924824a3fba6ea26f0a4b7916e706bb0d778a753a3b766088170641e6
/usr/share/ansible/files/ini_file   b85f3f3a51c9a3ce034ab7cddd8bd050d9568ff86c884fa8c6b6eeca664c470a
/usr/share/ansible/files/lineinfile     a3a833a5f8c5f68b658396223e53621305ad25a5d142f29f6ed87292dc4608fc
/usr/share/ansible/files/stat   eee4b2057912200bfcc0951a54b099bae97f42552b5ddcaf06cea2c29737da0a
/usr/share/ansible/files/synchronize    2a9d99831483db9b44b7a67d617304b7f535053a9f5aec38942b99702cf497fa
/usr/share/ansible/files/template   174e118aec6d4ced8405e047a50adeaa7c1c813452ad1f46a66c328bcf8bf838
/usr/share/ansible/files/unarchive  cd50332e4cd2fe3a15ab6a1ec9b7883255da8dca5b4df3d601dc6d029bbed016
/usr/share/ansible/files/xattr  82589448d050db04bb614a19159cfc207995329ea6e1c4da521c928e538d80e3
/usr/share/ansible/internal/async_status    1f14666dd3d4ce9cd0ed57f603d871689a6c5ac83704d960438fce28fffbb697
/usr/share/ansible/internal/async_wrapper   a0ad98ee9fabdc63ccbe5a8e712953c930438a66aa5c02cffb8f1eb1ccae8d67
/usr/share/ansible/inventory/add_host   98fb5fbfa5de7e36f6995de7ead04b23b129a0a6cde764230637387cdfed2088
/usr/share/ansible/inventory/group_by   868b4c13a3fcff2b72686fec3991c91f88ee178aa6aa5e0076b5828a744f14c4
/usr/share/ansible/messaging/rabbitmq_parameter     0c0b70106a24202db754ad37ccab16e80dd1a502b392dc937f7be4d5c8d5e378
/usr/share/ansible/messaging/rabbitmq_plugin    82b65ef821106550466b84eae4c13c8d343400764e3a1d7bd6a1e0ee57e589c9
/usr/share/ansible/messaging/rabbitmq_policy    039c643bf7343d50951f468470a60d671257962765d29b71f39ba4740aec2d9b
/usr/share/ansible/messaging/rabbitmq_user  ef024a403de02f160bddc2e1bb23cd67bd4e7889a0b76e080e717550db4ef998
/usr/share/ansible/messaging/rabbitmq_vhost     7b08114bc03d7f2fc53a6c3ce4a74859722338d740be15dd573ba0376d5102af
/usr/share/ansible/monitoring/airbrake_deployment   eacf50db646c64b3be7dd98d20853564168d61f1bcb21ae24e12c99f75284152
/usr/share/ansible/monitoring/boundary_meter    00d3991507838d98d7964c3309297dc08605075972f04a488ed87a34bdc09126
/usr/share/ansible/monitoring/datadog_event     85dc15669a9dc161a6aee47371d751e71bbc8da6e881915c3e030e8d5e78be65
/usr/share/ansible/monitoring/monit     25f0415b44055e95fd3656cff508a051e2156153ef52d27a51c356ecb1105e73
/usr/share/ansible/monitoring/nagios    dda13f1f0a09e4e88a3e72564543ed66d4196ee9fcce76c2ef2c28e7ff9a737a
/usr/share/ansible/monitoring/newrelic_deployment   98659c1c3234bc66031412f52891cd1ffbbce09db98eaa56b5428ba9cc527987
/usr/share/ansible/monitoring/pagerduty     d7edf7bb7ea82dbdeac29fccc9a8cd368d9cff14cf121f0a12af38bf2cc53405
/usr/share/ansible/monitoring/pingdom   7818a59f6672f11b7659562b64229d8a038b5f1307bfd1541d7caeb42b0206d9
/usr/share/ansible/net_infrastructure/arista_interface  d491b91dc4f290b1692403767e7db296a863a0af3665e2d60a72c927cfc16f59
/usr/share/ansible/net_infrastructure/arista_l2interface    8dbc0f2452d09e91a9a166868c1241b32816a6eabe81813294a289bcd4788b6d
/usr/share/ansible/net_infrastructure/arista_lag    5213902a0aad32a245ccab37758f9c39bc1770f370fb3e90d6f68176e4211be6
/usr/share/ansible/net_infrastructure/arista_vlan   3c85b3af946f8f7f3ad82724a6b55ab715258ccf65d80900d94233b4427f8ed2
/usr/share/ansible/net_infrastructure/bigip_monitor_http    677e3fde33709bd54705b407508cf49eaf61300231277c0a7a50c4745020a8fb
/usr/share/ansible/net_infrastructure/bigip_monitor_tcp     03a95b5ce48bd43dcbf3beb1a863309bc9c7f0b6318ed4b782eacfbcd9e108a0
/usr/share/ansible/net_infrastructure/bigip_node    c039b7fa2e7702c8fe75863ed73a291d671108cb4f7c5cfcf2e8fbd4c504ab76
/usr/share/ansible/net_infrastructure/bigip_pool    c9da0756dcf41010fc34f983d9d70d9ab02a88feded721385fd9aa317718ad72
/usr/share/ansible/net_infrastructure/bigip_pool_member     b6f5dbd1cb89e54719d52303a18a888a1be625405a5330a2831dfef2ea5301a8
/usr/share/ansible/net_infrastructure/dnsmadeeasy   9a8d3b016fb91e6253ec6fd871cbf310d37ef0c42b793c56fcbe973b63e89ef7
/usr/share/ansible/net_infrastructure/netscaler     a4c1ffa8655c6de2fdbc2a32ad846589d2866a98a9c94f78612e4498e5c97062
/usr/share/ansible/net_infrastructure/openvswitch_bridge    c07b7a392d5786f611ca34553b487f270ee07098c53feca8e9eeaa7effa77983
/usr/share/ansible/net_infrastructure/openvswitch_port  924a69624b681ee00091981a907fb23905218a0e5f7f5307c0157d3faf9c2eb2
/usr/share/ansible/network/get_url  4581afffc32f3d613ce230b0ab16fd40deb23091a502c9e8717b84ecce453e66
/usr/share/ansible/network/slurp    dca481b2961d3830f5ea7ce5b155586f5474fd64021e7b47545b279ac1d531b5
/usr/share/ansible/network/uri  203516eb1655b654fbf2019c913075a5ff4d8b511a3b0b42f207cb61a17c4721
/usr/share/ansible/notification/campfire    e65ad9e26b6686de4fe09521ebbd2adc3becb34cf5bbbe783eeeaa0f47e408c8
/usr/share/ansible/notification/flowdock    78a90a21823285d50e013888a8b0b985cbdaa4322a547da88f602307afdc5ff5
/usr/share/ansible/notification/grove   e19513224091d2b4bad5638ca7d4a960a464c1aa30b4e78367f9960fb4980ec0
/usr/share/ansible/notification/hipchat     0037b2f058236f063489b7952ae1aca98f2f3a2ebaa49a47dd6287f3ba6619aa
/usr/share/ansible/notification/irc     6d195d0f1d9679268ae181ce3963fc02e393655fe87368781765c31ab464f220
/usr/share/ansible/notification/jabber  f58b093a39e19c71790fb92aca9f7d6eb2ca467ff502d08531444ffa8cb15f07
/usr/share/ansible/notification/mail    dc3b329666043ea13fa03af0d2efaee1fdde0b1af0ff76d603d570867a2a8615
/usr/share/ansible/notification/mqtt    45dafdfe278c1f8030176f0ef897ec56001be7c6d23fff4d7d315b7563699920
/usr/share/ansible/notification/osx_say     975d9258cc8bf374772ca0760d5c3c2d988c54dc83b66f92b49e72848f4595c1
/usr/share/ansible/packaging/apt    e5e99c06ca280d0c41eb3e15aafb67556a401e9e55639919c43e13a903568a14
/usr/share/ansible/packaging/apt_key    4d51f7e48744b6fcb4de3721e131c6ea053cc1618ccc671efaba0a4bf494378b
/usr/share/ansible/packaging/apt_repository     951b9c71f88dc79618c1819a0ae853536e8e215cd8c1511ecb1123f8db3f756b
/usr/share/ansible/packaging/easy_install   893e555d3829e5e299a5ba9eba83184f2ecb94fcf0a18f63cf476cf6aabe6e3b
/usr/share/ansible/packaging/gem    b3bd712d842fbf1ad7a08ef421fdb395d07f8b64b628861eab56aa57ef6cd29d
/usr/share/ansible/packaging/homebrew   34b857a02e3935923c4f70824fe23fc794d9fd7196676d4a5bf6d752bccae826
/usr/share/ansible/packaging/macports   0d74054d76e2970d91c0a46ff2bd0ba28ee2921423c85dba92af2629ba57f44c
/usr/share/ansible/packaging/npm    a186ba8f8b021986c0de3f61d4ed8a3e51eacda7f83aec91513f660de45d6892
/usr/share/ansible/packaging/openbsd_pkg    18f11d9b843343bb86a6ae8c4241ac10607bb1ec94959a3f3cbf3be02b0e347f
/usr/share/ansible/packaging/opkg   606f956c0b4f57e9e1b6a7c16f05b89d7bcc39ef1ec993a36f82233a6eb9dcb6
/usr/share/ansible/packaging/pacman     aabe4615815cceb5450b810af17bf9cb31d58f3d2730690b02b6f9b8f2dedae6
/usr/share/ansible/packaging/pip    089c1b1f35ed2066b448747a8848cd4722342dfecd65e0d65497075da82745d9
/usr/share/ansible/packaging/pkgin  db9251a1b1e0cca58bdb9f9b69f79c5b20549d47130d698dc1926ee45df79de4
/usr/share/ansible/packaging/pkgng  2f8eea6e43b573e0f9bf3c33c93424b9a3684321e69ec73aef53391edd59aa19
/usr/share/ansible/packaging/pkgutil    84e9256c4407aab0b25739e2db44afc05ae161f91ede79505bc360eeed1b1974
/usr/share/ansible/packaging/portinstall    23ef5d76afdc00e80dce568a954d9e862da3123e9a41ebdebddb605b9eb6004b
/usr/share/ansible/packaging/redhat_subscription    3cb65d5fcdb0eee3d5d83dcd1276b1f78450dedb8883a37f0591db91be00f7b7
/usr/share/ansible/packaging/rhn_channel    be19bfd7d1dd6c34b5bea4ea5d2348080a6f124613b96981118b6dee8d6b2dbb
/usr/share/ansible/packaging/rhn_register   92ae6e6925de0589d403f12e19a87974daaccf6bd2f9d47d680991df9c3c2dc1
/usr/share/ansible/packaging/rpm_key    a9c976dd6e5983b9821fe0a8b9e2faaa1ab2eafca294af7499369088e9bdc6b6
/usr/share/ansible/packaging/svr4pkg    2cc563d9d61347a29463957fbe9bad1331baa0ee9e677ff4564c68354d696333
/usr/share/ansible/packaging/swdepot    598050d65f829069f320b741997b3c3c2636838453b8806fdce6e45dee64d8ec
/usr/share/ansible/packaging/urpmi  50f16b7a93258c3c7d9ba0343867b5c73e5264e738c247e39512c97a4525b4fe
/usr/share/ansible/packaging/yum    7e35d5c0110a9244e1f6b1afc9427ea3ed5badd284aee3c4045daaf3551c04b7
/usr/share/ansible/packaging/zypper     365c1eff47271159d04ccff3d6968e7472c2d8dfffe8a6ae6a43e7686b11df9e
/usr/share/ansible/packaging/zypper_repository  71507ecc9b57f173e5dcc0dcd62d7e606aa386a4d8374f58d70ca8c2251a4880
/usr/share/ansible/source_control/bzr   64b8dff08ab8de263479506309f46658a0d71d8ddd39db0fb9594fe1ee18ec68
/usr/share/ansible/source_control/git   5f022cf25d3bc9b546fc15c9ef55e308ae269f9bb6cd9adb7e036f8e2a5ffd44
/usr/share/ansible/source_control/github_hooks  ddee43c91003d60002da7586224941404a1e1217f5e5801317c1a731bb97d446
/usr/share/ansible/source_control/hg    98b13da79fda931f20820218c12950d8916d1540f6cacd05db57fbfb942132a6
/usr/share/ansible/source_control/subversion    8d6261d59f150d25499442c51fcf457809c50703ceead65e5f5c3cc34ce36653
/usr/share/ansible/system/at    298ef412dc9f96588a0942c393286c838420b1ddb1d708728871472831a34ec6
/usr/share/ansible/system/authorized_key    35baffcd1078cb9f3f79f59e5832cbed2e7ca568c42e17badbee0020d1e5533f
/usr/share/ansible/system/cron  e13b66ae9681c1837411596c4d92d0257b6cd0f7b3cd0b9781720da03ed9aafa
/usr/share/ansible/system/facter    af6655ac0a4d36ee485aa7f0e71a3c31f7c70f793b1061de276a7330a5e1b8c5
/usr/share/ansible/system/filesystem    c5f4282788080c8236bc1b289040f97eaceb84c34e1d4d8f2fa91aad1c638606
/usr/share/ansible/system/firewalld     90489430e84b5e10a02f0f01001fd7de310d4a683919b7866b8f1c0dadc09bdf
/usr/share/ansible/system/group     e5606b945a35b97876f990a12c2ff0eda27f6c586e4f1d80f7c77eac8d778d13
/usr/share/ansible/system/hostname  5745cc8cacccfafdfb8d0ccc6f4f6084e5e5d94ad9a5585ba5e75ac73ed8d51d
/usr/share/ansible/system/kernel_blacklist  5b4469079ad2e82385f66cbdefb2a12882db2336b69aa16bc2554a15003a9a51
/usr/share/ansible/system/lvg   9cff510ecb85ff69c496f4dfb7dbfe36b2516c1c83d2810e112a049a4321ac3b
/usr/share/ansible/system/lvol  6dbe76275abfec1494e9edabd549537b280f5142e553f98f987a737b38bde012
/usr/share/ansible/system/modprobe  5be12be44f267322708168239d7f814cf21f06ed1bf60e9e554c746790847f3d
/usr/share/ansible/system/mount     20a87c909bbdc69026f9774c188c4ab41be7388e7492b0e46d6cd3c344f3b729
/usr/share/ansible/system/ohai  81bf2ab5ec5234130383918aa027b941ee8f73a42ed12a2f2862dcc3756c7274
/usr/share/ansible/system/open_iscsi    05f54b349ee9fb7c6a4dbd3164ae37ea7861d4e1939ee736eaf82f8618b3e352
/usr/share/ansible/system/ping  21e73e14af3c2e37d16a8800d863d7dd267ee54317123d7b5d8eff91d84f3633
/usr/share/ansible/system/seboolean     fda0f74297cf5b4c664a34c2035a35d9ca028a35c8d8bb6b97a42a35241ed9eb
/usr/share/ansible/system/selinux   7917f365067dcbbb104899e292608b2737afb41506b1e5191b218416b88beed2
/usr/share/ansible/system/service   3fb2f418e333674ed3c07d118097ca58a1b698f4427255a698859c6d57e53125
/usr/share/ansible/system/setup     3a6a23de31439a25aafc5143d98ad5aa0a4397159275e31d70ad028dee44dd96
/usr/share/ansible/system/sysctl    3d60018c39385a58f5ff3fbc8e15af75311a5cccae96a4239aaab395e049bec3
/usr/share/ansible/system/user  90c5061cd61bee1b5b0ce5f8b1dd8c4361ed82e0eda4e8a094cdf8d1766759a2
/usr/share/ansible/system/zfs   afc95da3a5a3350fb420d8af2ef2f71801ba7c177a57e114854365fcea670fe2
/usr/share/ansible/utilities/accelerate     34391bf3f0ca65467ee696b07e3a18bbdd638ca694de10c74a820fb075f7e56c
/usr/share/ansible/utilities/assert     0b7838286752ddf936d8efe78ba7b2a9e1129766220b8b54ad97c582860bc9c9
/usr/share/ansible/utilities/debug  3dff2b9b2b2fc7953330cdc3a8686606b9e7cabe10fa7a044d39c8545545499d
/usr/share/ansible/utilities/fail   eff18bf94ffba609eea9fa122a0ffbee6b57617e2d04db81a2fe63d248e0ef59
/usr/share/ansible/utilities/fireball   0facd8c1d5db0e39f0b02619eaaeb2f56fc3e544690d160f8f72d5d2d3d3297d
/usr/share/ansible/utilities/include_vars   b19c699cd36ba05bdb64dce5579b9a8bf6c570f3c145cda8ea0f03c8cd3e1325
/usr/share/ansible/utilities/pause  926b94da0a071006bd2d746e02a5f3a239ce60d373eccb51b326a6ef953a3e3b
/usr/share/ansible/utilities/set_fact   11ca6cb9bf9c6cbf8e7106a77a5d7bdaeb01233412b8d32bcc9ed85ad4439f08
/usr/share/ansible/utilities/wait_for   7cda7c4fc8eb9f2f5a96be7af02d772a431104c234ba09424d31c18c6d40c594
/usr/share/ansible/web_infrastructure/django_manage     a00d586a4fd83085007ef4ad8a42c8a25a17bc7827124c59fe1b4b439340b426
/usr/share/ansible/web_infrastructure/ejabberd_user     03f658eead78b39916db483c392b6d3d5b131e9ccbc87030b8b5e9406262adab
/usr/share/ansible/web_infrastructure/htpasswd  968b7f686c00e994e17fb319d8859b4d36bdd092f6254ce5b4583df343c882aa
/usr/share/ansible/web_infrastructure/jboss     38c32ea54e5fc291e0a263e78a09175a01e57419c98e89dfa1d2645c0f700080
/usr/share/ansible/web_infrastructure/supervisorctl     818a3064cc6a22a0d3f54a9d80abe663123fa167cdc67a9ec05e0daea59cb691
```

**Comment by jctanner** [comment]
@evanccopengeo it looks like you have the right file for 1.5.3. The problem seems to be that the "source_dest_check" parameter was merged in on March 11th by 5d4980459ff02f604b0931a235209e744e08992f which was after the 1.5.0 release. All 1.5.x releases afterwards have been critical bugfixes and security updates, so this particular parameter has never existed in 1.5.x. I will update the docstrings in the module to say it was introduced in 1.6. Sorry about the confusion.
